backend/src/api/v1/auth/routes.py

```diff
@@ -68,3 +71,33 @@ async def confirm_user_verification(token: str, session: TransactionalSession) -
     verify_user(session, user)
 
     return {"detail": "Email verified"}
+
+
+@router.post("/password-recovery")
+async def recovery_password(schema: UserEmail, session: TransactionalSession) -> Any:
+    user = get_user_by_email(session, schema.email)
+
+    if not user:
+        raise HTTPException(status.HTTP_404_NOT_FOUND, "User not found")
+    if not user.is_active:
+        raise HTTPException(status.HTTP_422_UNPROCESSABLE_ENTITY, "User isn't active")
+    if user.is_verified:
+        raise HTTPException(status.HTTP_422_UNPROCESSABLE_ENTITY, "Email already verified")
+
+    await send_reset_password_message(schema.email)
+
+    return {"detail": "Email has been sent"}
+
+
+@router.post("/reset-password")
```

Comment:
since this is a request for editing a part of a user, PATCH is needed
```suggestion
@router.patch("/reset-password")
```